openedx/frontend-app-publisher
# feat: add display on org page field in AdditionalMetadata

### [PROD-3481](https://2u-internal.atlassian.net/browse/PROD-3481)

### Description
Add display on org page checkmark under AdditionalMetadata fields (only displayed for ExecEd types). Since the field is boolean, there is no need to update the config `ADDITIONAL_METADATA_REQUIRED_FIELDS`. 

Related to PR https://github.com/openedx/course-discovery/pull/4021

### Testing
- Pull latest discovery and run migrations. Ensure you have one ExecEd 2u type course
- Checkout this branch for publisher
- Run publisher and visit the execEd course
- You should see the checkmark right under Additional Metadata section
- Toggle the value and ensure the backend is updated as expected

### Screenshots

<img width="1335" alt="Screenshot 2023-07-25 at 12 54 12 PM" src="https://github.com/openedx/frontend-app-publisher/assets/40599381/efa811cd-a6e2-4a90-9c02-5cce5eff56d6">


<img width="1335" alt="Screenshot 2023-07-25 at 12 59 32 PM" src="https://github.com/openedx/frontend-app-publisher/assets/40599381/d48898ef-15f6-4f29-a3f0-2c1c589e0c2d">
